src/lib/dev.ts

```diff
@@ -96,9 +117,15 @@ function createPageRenderer(appRoot, jsonModules, pageName, onInit, onUpdate, on
   };
 }
 
-function registerComponent(path, content, options) {
-  const map = options.registerPartialMap || [
-    path => (path.includes('/block/') ? /\/([^/]+)\.hbs/gi.exec(path)[1] : null),
+function registerComponent(path: string, content: any, options: BootstrapOptions) {
+  const map: Array<(path: string) => string | null> = options.registerPartialMap || [
```

Comment:
hmm, this change makes it a bit verbose, since the same construct could also be used from user code that will do multiple of those inline items in a row. Not sure if there is a better option though...

Reply:
Can you elaborate? Not sure I understand. I can leave out the type annotation on this line because the compiler can implicitly infer it, thought that will not change anything about the typing.